### ISSUE TITLE
Error during installation of some packages

### ISSUE THREAD

**Issue by clb92** [ich777/un-get#2]
For some packages, I get this error in the output:

```
# un-get install libewf
The following package(s) will be installed: libewf
Are you sure? [y/N] y
/boot/extra/: Is a directory
rm: cannot remove '/boot/extra/': Is a directory

Can't download package libewf, please try again later...!
```

I get the same thing with several other packages, like `parallel`, `md5deep` and `p7zip`, just to name some.

**Comment by ich777** [comment]
@clb92 thank you for the report, released v0.14 or 2022.09.25 respectively and this should be fixed.

If you experience any issues feel free to open up the issue again or create a new issue.

**Comment by clb92** [comment]
Thank you for the quick fix.
